src/components/instantlaunches/launch/index.js

```diff
@@ -81,14 +86,24 @@ const InstantLaunchStandalone = (props) => {
         isFetchingUsageSummary,
     ]);
 
+    useEffect(() => {
+        if (error || resourceError) {
+            setDialogError(error || resourceError);
+            setErrDialogOpen(true);
+        }
+    }, [error, resourceError, setErrDialogOpen, setDialogError]);
+
     if (isLoading) {
         return <LoadingAnimation />;
     } else if (error || resourceError) {
-        const err = error || resourceError;
         return (
-            <ErrorTypographyWithDialog
-                errorMessage={t("instantLaunchError")}
-                errorObject={err}
+            <DEErrorDialog
+                open={errDialogOpen}
+                errorObject={dialogError}
+                handleClose={() => {
+                    setErrDialogOpen(false);
+                    router.push("/");
+                }}
```

Comment:
My question here is basically: Should I have the `DEErrorDialog` (closed) always get rendered in and just let the `useEffect` open it, or is it still worth having this `else if` to only render in the dialog when we actually have an error?

Reply:
(er, I guess by "always get rendered in" I mean after the loading animation is finished, probably still shouldn't render it on the load screen)

Reply:
> My question here is basically: Should I have the `DEErrorDialog` (closed) always get rendered in and just let the `useEffect` open it, or is it still worth having this `else if` to only render in the dialog when we actually have an error?

In this case [I don't think the `useEffect` is needed at all](https://react.dev/learn/you-might-not-need-an-effect), since rendering the dialog is already conditional on the `isLoading` and error states.
```jsx
            <DEErrorDialog
                open={true}
                errorObject={error || resourceError}
                handleClose={() => {
                    router.push("/");
                }}
            />
```

With that said, I think it might be simpler to use the `src/components/error/WrappedErrorHandler`, for example:
https://github.com/cyverse-de/sonora/blob/b26b43437d8dddd1397d7132e4627e1d07901457/src/components/apps/launch/index.js#L138